hyperspy/docstrings/signal1d.py

```diff
@@ -48,7 +48,9 @@
         """
 
 SPIKES_REMOVAL_TOOL_DOCSTRING =\
-        """Graphical interface to remove spikes from EELS spectra.
+        """Graphical interface to remove spikes from EELS spectra. 
```

Comment:
not restricted to EELS, just give EELS as example or maybe even add luminescence spectra as second example?